src/main/java/it/auties/whatsapp/controller/DefaultControllerSerializer.java

```diff
@@ -170,7 +170,9 @@ public void serializeStore(Store store, boolean async) {
     }
 
     private CompletableFuture<Void> serializeChat(Store store, Chat chat) {
-        var path = getSessionFile(store, "%s%s.smile".formatted(CHAT_PREFIX, chat.jid().toString()));
+        String fileName = "%s%s.smile".formatted(CHAT_PREFIX, chat.jid().toString());
+        fileName = fileName.replaceAll(":", "~~");
```

Comment:
Move this logic to getSessionFile
Also we might want to use some standardized method to escape the path instead of handling the specific case

Reply:
The issue in character ':'. It is not allowed in the filename.

chat.jid().toString() - this produces contact string with ':' char